mantidproject/mantid
# Plot focused workspaces via check-box

Fixes #13502

**To Test**
Go to `Interface/Diffraction/Engineering Diffraction`
To use the focus tab: first you need to calibrate, following the steps
explained in #13370 (you need a few files too). If you've done it
before the GUI will remember the last settings. 

Once you have done that, then go to the focus tab, and give:
- run number: you can try 228061 (the file ENGINX00228061.nxs is available from the unit test data)
  and/or 213855 (the file ENGINX00213855.nxs is available from the doc test data), or other ENGINX runs from the archive.
- Bank: select 1 or 2
- Click `Focus`

_You should not see  `engggui_focusing_output_ws`  workspaces in the workspaces window_.
- Go back on `Engineering Diffraction Analysis` Interface and **tick** `Plot Focused Workspace` from the `Output` section
- Click `Focus`

_Should plot spectrum graph for `engggui_focusing_output_ws` workspace._
